xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -166,8 +166,8 @@ The examined data communicates to the pipeline when it should signal more data i
       </Parameters>
       <Docs>
         <param name="leaveOpen">An optional flag that indicates whether disposing the returned <see cref="T:System.IO.Stream" /> leaves <see cref="T:System.IO.Pipelines.PipeReader" /> open (<see langword="true" />) or completes <see cref="T:System.IO.Pipelines.PipeReader" /> (<see langword="false" />).</param>
-        <summary>Returns the underlying <see cref="T:System.IO.Stream" /> wrapped by the <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
-        <returns>The underlying stream.</returns>
+        <summary>Returns a <see cref="T:System.IO.Stream" /> representation of the <see cref="T:System.IO.Pipelines.PipeReader" />.</summary>
+        <returns>A <see cref="T:System.IO.Stream" />.</returns>
```

Comment:
```suggestion
        <returns>The underlying stream.</returns>
```

The return type is added automatically, so we aren't supposed to include it. See https://github.com/dotnet/dotnet-api-docs/wiki/Returns:-Methods.